Readme.md

```diff
@@ -56,6 +56,8 @@ var cardPaymentModel = new CardPaymentModel
 	
 };
 ```
+**Note:** Please make sure that you are using a unique Consumer Reference for each different consumer.
```

Comment:
Could I suggest re-phrasing as "Please make sure the Consumer Reference you supply uniquely identifies each customer." Otherwise we'll get people who will just generate a unique guid each time then wonder why it doesn't work.